modules/publisher/api.py

```diff
@@ -91,16 +90,7 @@ def get_authorization_header(session):
 
 
 def get_account(session):
-    authorization = authentication.get_authorization_header(
-        session['macaroon_root'],
-        session['macaroon_discharge']
-    )
-
-    headers = {
-        'X-Ubuntu-Series': '16',
```

Comment:
I'm guessing we don't need the X-Ubuntu headers anymore?

Reply:
nope, here we are requiring the account information this is not needed